### PR TITLE
feat: added impetus007 as contributor

### PR DESCRIPTION
**Pull request by impetus007**

### CODE REVIEW

**Comment by Burnleydev1** [comment]
image
Please can take look at this, making sure you do not remove any existing code

File: Task1/intro.json [hunk 1]
```diff
@@ -174,12 +174,6 @@
     "hacktoberfest_round": "first",
     "role": ["contributor"]
   },
-  {
```
Comment (Burnleydev1) [inline Task1/intro.json:177]: Can you please put this back in the code?
Or update your branch?

File: Task1/intro.json [hunk 2]
```diff
@@ -174,12 +174,6 @@
     "hacktoberfest_round": "first",
     "role": ["contributor"]
   },
-  {
-   "name": "Lando Momo Paul",
-   "github_handle": "lando-paul",
-   "hacktoberfest_round": "first"
-   "role": ["contributor"]
-  },
```
Comment (Burnleydev1) [inline Task1/intro.json:182]: starting from there spanning to this line

**Review by Burnleydev1 — CHANGES_REQUESTED** [review]

File: Task1/intro.json (hunk 1 above)
Comment (impetus007) [reply Task1/intro.json:177]: sure

**Review by impetus007 — COMMENTED** [review]
Changes Done

**Comment by Burnleydev1** [comment]
The changes were no reflected from my end

**Review by Burnleydev1 — APPROVED** [review]

**Pull request merged**